bundles/org.openhab.binding.freeathomesystem/README.md

```diff
@@ -0,0 +1,150 @@
+# OpenHAB 3 ABB/Busch-free@home Smart Home binding
```

Comment:
1. It is meant to be openHAB, note the lower case o at start.
2. Please don't refer to a version number as this will out date when V4 is released.

```suggestion
# ABB/Busch-free@home Smart Home binding
```